test/SparqlAntlrParserTestHelpers.h

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Robin Textor-Falconi (textorr@informatik.uni-freiburg.de)
+
+#pragma once
+
+#include <gmock/gmock.h>
+
+#include "../src/parser/data/VarOrTerm.h"
+
+template <class>
+constexpr bool always_false_v = false;
+
+std::ostream& operator<<(std::ostream& out, const GraphTerm& graphTerm) {
+  std::visit(
+      [&](const auto& object) {
+        using T = std::decay_t<decltype(object)>;
+        if constexpr (std::is_same_v<T, Literal>) {

```

Comment:
In some Header I have
`ad_utility::isSimilar<T, Literal>`

`[&] <typename T>(const T& object)`

Or:
The decay_t here is not necessary, since you are using a `const&` and not a universal `&&`.